docs/advanced.rst

```diff
@@ -758,6 +758,35 @@ Please refer to the supplemental example for details.
     length queries (``__len__``), iterators (``__iter__``), the slicing
     protocol and other kinds of useful operations.
 
+C++11 chrono datatypes
+======================
+When including the additional header file :file:`pybind11/chrono.h` conversions from c++11 chrono datatypes
+to corresponding python datetime objects are automatically enabled.
```

Comment:
Super minor, but it would be nice to rewrap these text blocks to 80 characters, like the rest of the documentation (I tend to read `advanced.rst` directly more often than I look at generated HTML).  If that's a nuisance in your editor, don't bother (but consider looking for a new editor ;-) ).